javatests/arcs/android/systemhealth/testapp/StorageCore.kt

```diff
@@ -137,7 +143,7 @@ class StorageCore(val context: Context, val lifecycle: Lifecycle) {
 
         if (settings.function != Function.STOP && settings.timesOfIterations > 0) {
             tasks = Array(settings.numOfListenerThreads + settings.numOfWriterThreads) { id ->
-                object : ScheduledThreadPoolExecutor(1) {
+                object : ScheduledThreadPoolExecutor(2) {
```

Comment:
Would you mind changing a little bit: 
for performance test, use pool size 1 as this is required to stress some situations like how good/bad of back-pressure.
for stability test, use pool size 2 as we need an additional trouble-maker.

ScheduledThreadPoolExecutor (if (settings.function == Function.STABILITY_TEST) 2 else 1)

Reply:
Sure.